AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -313,11 +311,11 @@ - (void)setEnabled:(BOOL)isEnabled {
   self.enabledStateUpdating = YES;
   if ([self isEnabled] != isEnabled) {
 
-    // Enable/disable pipeline.
```

Comment:
Any intentions of this change?

Reply:
Because `isEnabled` is checked in `applyPipelineEnabledState` (`sendStartServiceLog`)

Reply:
The `isEnabled` is a parameter passed to `applyPipelineEnabledState`. It doesn't read anything in NSUserDefaults. I think we should make sure the pipeline is properly changed before setting the value to NSUserDefaults.

Reply:
Changing this is not a big deal but I would like to keep the previous code to get rid of a chance for side effect. Please revert unless it has to be changed.

Reply:
OK I got it. `isEnabled` from `sendStartServiceLog`.